src/platform_impl/windows/event_loop.rs

```diff
@@ -265,10 +265,37 @@ impl<T> EventLoopWindowTarget<T> {
     }
 }
 
+/// Returns the id of the main thread.
+///
+/// Windows has no real API to check if the current executing thread is the "main thread" unlike macOS.
+/// Unix-like OSes have some APIs or semantics that exist regarding whether the main thread is in use or
+/// if the main thread id equals the pid like on Linux.
```

Comment:
Could you clarify a bit here: is there a Unix API to check for the main thread, or is it guaranteed that "main thread id = pid"?

Reply:
Decided to omit the Unix example since each unix has it's own way of checking for the main thread.